### ISSUE TITLE
Announcement: Planned Deprecation of the v6 cf CLI

### ISSUE THREAD

**Issue by chenl23** [chenl23/cfar-proposals#13]
### Proposal Name
Announcement: Planned Deprecation of the v6 cf CLI

### TL;DR Summary
Following on the CC V2 API Deprecation Plan notification recently, the cf CLI team are writing to also announce the planned deprecation of the v6 cf CLI. Please provide feedback and review the v6 CF CLI Deprecation Plan for details on strategy and timeframes. 

We hope to make the upgrade process as seamless as possible for our users, and that the v7 cf CLI will provide greater opportunities to grow and enhance the product for our users.

As always, we value your feedback and would appreciate comments in the document. We also hang out on Cloud Foundry Slack at #cli. We hope to hear from you. 

### Proposal URL
https://docs.google.com/document/d/1kX-UbQzz7y41sgK979Zk_IHISQwfSH0vBT27mIeqo5Q/edit#

### Point of Contact
Abby Chau
